victoriensukarieh.Calculator/ConsoleCalculator/Program.cs

```diff
@@ -0,0 +1,181 @@
+﻿using CalculatorLibrary;
+using ConsoleCalculator.Models;
```

Comment:
Please remove unecessary usings and add spacing between usings and namespace, and between namespace and class name, as per code conventions: 

![image](https://user-images.githubusercontent.com/108711476/228714775-f9c5f016-a425-4e8f-8cde-0a0d2a9c9f85.png)